modules/installation-custom-aws-vpc.adoc

```diff
@@ -4,6 +4,13 @@
 // * installing/installing_aws/installing-aws-private.adoc
 // * installing/installing_aws/installing-aws-vpc.adoc
 
+ifeval::["{context}" == "installing-aws-china-region"]
+:aws-china:
+endif::[]
```

Comment:
@bmcelvee The `ifeval` for aws china does not apply to 4.8. Please remove it and the closing statement.

Reply:
Done!